DESCRIPTION

```diff
@@ -13,14 +13,14 @@ Authors@R: c(
 	comment = c(ORCID = "0000-0001-7109-2668")),
     person(given = "Pedro", family = "Larrañaga", role = "ths",
 	comment = c(ORCID = "0000-0003-0652-9872")))
-Description: Simulation of positive definite matrices constrained by an undirected or acyclic directed graph structure.
-License: GPL (>=2) 
+Description: Simulation of correlation matrices possibly constrained by a Gaussian graphical model. In particular, the package provides functions that implement the simulation methods described in Córdoba et al. (2018a) <doi:10.1007/978-3-030-03493-1_13>, Córdoba et al. (2018b) <doi:10.1145/2695664.2695717> and Córdoba et al. (2019) <arXiv:1909.01062>.
```

Comment:
Simulation of correalation matrices, possibly with sparsity pattern compatible with a given graph.  
(...constrained by a Gaussian graphical model, can be misleading)